### DISCUSSION TITLE
Какая настройка делает вот это?..

### DISCUSSION THREAD

**Discussion by Darkkys** [OGSR/OGSR-Engine#389] [Q&A] [answered]
![S T A L K E R   Call of Pripyat Screenshot 2022 01 08 - 13 16 03 60](https://user-images.githubusercontent.com/97330663/148634392-024b470c-1819-439f-8749-1b7d1209d540.png)

Логичные тесселяция\параллакс не отключают "игольчатость" террейнов...

**Comment by I-AmDead** [comment, marked as answer]
> ![S T A L K E R Call of Pripyat Screenshot 2022 01 08 - 13 16 03 60](https://user-images.githubusercontent.com/97330663/148634392-024b470c-1819-439f-8749-1b7d1209d540.png)
> 
> Логичные тесселяция\параллакс не отключают "игольчатость" террейнов...

Это паралакс на террейне, для нормального вида нужны хорошие бамп текстуры.

**Reply by Darkkys** [reply]
Отключить это можно? )

**Reply by xrSimpAI** [reply]
https://github.com/OGSR/OGSR-Engine/blob/a3ae41c6148c4496f58d2bec6417f96a9c4756ac/Game/Resources_SoC_1.0006/gamedata/shaders/r3/deffer_impl_flat_d.ps#L4
эту строку закомментировать, две выше - раскомментировать.
Но лучше текстуры поменять, да. Хотя найти их сложно, а делать качественные текстуры с бампами почти никто не умеет.(

**Reply by Darkkys** [reply]
вроде с "фотореалистик зона" более менее работает
